### ISSUE TITLE
Why need tanh for the loss?

### ISSUE THREAD

**Issue by Aiolus-X** [belkakari/implicit-kan#2]
Hallo! In the code, the loss is computed as: loss = ((F.tanh(out) - img) ** 2).mean(), why it needs tanh?

**Comment by belkakari** [comment]
Hey! I don't think it's needed but this is one of the decisions I made to keep the output in [-1, 1] range. It works without tanh too

**Comment by Aiolus-X** [comment]
Thank you for the reply!
